ext/session/tests/bug10807.phpt

```diff
@@ -0,0 +1,69 @@
+--TEST--
+Bug #10807 (session_regenerate_id with custom handler and use_strict_mode generates three new session ids)
```

Comment:
I agree this is a bit confusing at first... But "Bug #XXXXX" is for the old bugs.php.net bug IDs. For GitHub we use "GH-XXXXX". So in this case you should change `Bug #10807` to `GH-10807`.
Similarly, you should name your file `gh10807` instead of `bug10807`.

Reply:
Changes made. Might I suggest that the docs at https://qa.php.net/write-test.php are updated to handle this.

Re rebasing to the 8.1 branch, my fork doesn't have this branch and this has exceeded my github knowledge. Are you able to take this over for now?